modules/apps/data-engine/data-engine-api/bnd.bnd

```diff
@@ -5,6 +5,7 @@ Export-Package:\
 	com.liferay.data.engine.content.type,\
 	com.liferay.data.engine.exception,\
 	com.liferay.data.engine.field.type.util,\
+	com.liferay.data.engine.intrinsic.object,\
```

Comment:
it can't be native because "native"is a java keyword. I would have to concat the package name to something like: ".....data.engine.nativeobject" . Isn't that weird?